tests/test_maze.c

```diff
@@ -3,7 +3,7 @@
 int main() {
 
     printf("Maze creation test\n");
-    Maze * maze = create_maze("maze/");
+    Maze * maze = create_maze("maze/", 0, 0);
```

Comment:
Ajouter un commentaire comme `Create the maze from folder maze/ and initialise the player to coordinate (0;0)`.